src/locales/en.json

```
@@ -2897,6 +2897,7 @@
 "message.kubernetes.cluster.start": "Please confirm that you want to start the cluster",
 "message.kubernetes.cluster.stop": "Please confirm that you want to stop the cluster",
 "message.kubernetes.cluster.upgrade": "Please select new kubernetes version",
```

```suggestion
"message.kubernetes.cluster.upgrade": "Please select new Kubernetes version",
```
@davidjumani can you please change this as well